### DISCUSSION TITLE
How to detect both Text and Tables from PDF successfully with PaddleOCR and write to Excel?

### DISCUSSION THREAD

**Discussion by goapurva** [PaddlePaddle/PaddleOCR#10544] [Q&A]
Hello,

I am working with PDF files that have text as well as tables. I would like to detect the horizontal text and tables and write it to an excel file for manipulating the data in the tables. However, the text above and below the tables is also very important and needs to be extracted and written to the same excel file.

I am currently using the version of PaddleOCR that comes bundled with img2table. However, the text is not being extracted and some table data is also missed.

I would appreciate some help and examples for the above, directly using PaddleOCR.

Thanks,
Apurva

**Comment by CapturefastCEO** [comment]
Try https://BeyondOCR.com API for table detection and extraction free up to 250 pages.
